### PR TITLE
Add OpenShift templates

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: openshift/imageStream-template.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: revsolver-imagestream
+  annotations:
+    description: This is Thoth Core - Reverse Solver ImageStream
+    openshift.io/display-name: "Thoth: Reverse Solver ImageStream"
+    version: 0.4.0
```
Comment (pacospace) [inline openshift/imageStream-template.yaml]: Why different versions in these template?

**Review by pacospace — COMMENTED** [review]
One small comment, otherwise LGTM!

File: openshift/imageStream-template.yaml (hunk 1 above)
Comment (fridex) [reply openshift/imageStream-template.yaml]: copy-:spaghetti: error

**Review by pacospace — APPROVED** [review]
LGTM! Thanks :pizza:

**Pull request merged**